sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -334,6 +334,143 @@ public class SampleApplication implements CommandLineRunner {
 ```
 Autowired UserRepository interface, then can do save, delete and find operations. Spring Data Azure Cosmos DB uses the CosmosTemplate to execute the queries behind *find*, *save* methods. You can use the template yourself for more complex queries.
 
+## Support multi-database configuration
+The `azure-spring-data-cosmos` support multi-database configuration, here is a sample to config multiple account database
+
+### Add the dependency
+[//]: # ({x-version-update-start;com.azure:azure-spring-data-cosmos;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-spring-data-cosmos</artifactId>
+    <version>3.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Config application properties
+The example uses the `application.properties` file
+```properties
+azure.cosmosdb.primary.uri=your-primary-cosmosDb-uri
+azure.cosmosdb.primary.key=your-primary-cosmosDb-key
+azure.cosmosdb.primary.secondaryKey=your-primary-cosmosDb-secondary-key
+azure.cosmosdb.primary.database=your-primary-cosmosDb-dbName
+azure.cosmosdb.primary.populateQueryMetrics=if-populate-query-metrics
+
+azure.cosmosdb.secondary.uri=your-secondary-cosmosDb-uri
+azure.cosmosdb.secondary.key=your-secondary-cosmosDb-key
+azure.cosmosdb.secondary.secondaryKey=your-secondary-cosmosDb-secondary-key
+azure.cosmosdb.secondary.database=your-secondary-cosmosDb-dbName
+azure.cosmosdb.secondary.populateQueryMetrics=if-populate-query-metrics
```

Comment:
As part of rebranding, we are updating cosmosdb -> cosmos . That's one of the changes I did in my other PR. 

Please change these to `azure.cosmos....` instead of `azure.cosmosdb...`